src/query.rs

```diff
@@ -1009,7 +1009,7 @@ add_lints!(
     constructible_struct_adds_field,
     constructible_struct_adds_private_field,
     constructible_struct_changed_type,
-    struct_marked_non_exhaustive_changed_type,
+    non_exhaustive_struct_changed_type,
```

Comment:
Please keep these sorted, or else our automation will break.

Reply:
See the comment above the macro

Reply:
Good catch